dialog/dialog/pipelines.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+# Define your item pipelines here
+#
+# Don't forget to add your pipeline to the ITEM_PIPELINES setting
+# See: http://doc.scrapy.org/en/latest/topics/item-pipeline.html
+
+import os
+import requests
+
+API_URL = 'http://czl-api.code4.ro/api/publications/'
+API_TOKEN = os.environ['API_TOKEN']
+
+class DialogPipeline(object):
+
+    def process_item(self, item, spider):
+        self.upload(item)
```

Comment:
One thing that might be useful is to add an intermediate pipeline that checks an item has the required metadata, discarding items that do not. This will reduce the total number of requests that will fail.